android/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.2'
+        classpath 'com.android.tools.build:gradle:8.2.0'
```

Comment:
While this shouldn't cause any issues, was this needed? So we can extract to a different PR (In purchases Android we currently have 8.1.1)

Reply:
Same, I reverted this in my followup commits